Sources/SPTPersistentCache.m

```diff
@@ -22,36 +22,36 @@
 
 #import <SPTPersistentCache/SPTPersistentCacheHeader.h>
 
-#import "crc32iso3309.h"
 #import "SPTPersistentCacheRecord+Private.h"
 #import "SPTPersistentCacheResponse+Private.h"
 #import "SPTPersistentCache+Private.h"
 #import "SPTPersistentCacheGarbageCollector.h"
 #import "NSError+SPTPersistentCacheDomainErrors.h"
 #import "SPTPersistentCacheFileManager.h"
-#include <sys/stat.h>
 #import "SPTPersistentCacheTypeUtilities.h"
 #import "SPTPersistentCacheDebugUtilities.h"
 #import "SPTPersistentCachePosixWrapper.h"
 
+#include <sys/stat.h>
+
+#include "crc32ios3309.h"
+
 // Enable for more precise logging
 //#define DEBUG_OUTPUT_ENABLED
 
-NSString *const SPTPersistentCacheErrorDomain = @"persistent.cache.error";
-static const uint64_t kTTLUpperBoundInSec = 86400 * 31 * 2;
+typedef SPTPersistentCacheResponse* (^FileProcessingBlockType)(int filedes);
+typedef void (^RecordHeaderGetCallbackType)(SPTPersistentCacheRecordHeader *header);
```

Comment:
Since you’re fixing style issues. This symbol should have at least the `SPT` prefix. Potentially `SPTPersistentCache` even.